### PR TITLE
Mergin feature branch into master branch

### PR DESCRIPTION
**Pull request by guillainbisimwa**
- created linters 
- Used Bootstrap
- Added Header, body and footer

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED ♻️ 

Good job so far team!! 👏 
Your project is almost a clone! I am impressed! 👍 
Good job setting up bootstrap!!
You have descriptive commits and the readme is pretty good! Everything very professional! Congratulations! 🥇 
Good job with the linters as well 🟢 

However, there are some changes required before approving this project and move forward!
Check my comments below 👇 

Kindly make the changes and once done resubmit your project for a new code review 📑 

If you have any doubt or question, you can contact me on Slack 24/7

Good luck with the changes 😃 

- Navbar
   - The only fixed part of the navbar should be the red one. In big screens, your second navbar ( the white one ) is also fixed when it shouldn't. 
   - The navbar should always fit the maximum width of the screen, in your case, you have some padding that is not allowing the navbar to fit all the width. Please, fix it.

- Screens below 798px
   - The order of the sections should be: **top story, featured stories, debate,  opinion, more stories, my turn, culture and travel**. What you got is: **top story, my turn, more stories, featured stories, debate, opinion**. Kindly position the sections in the right order. 
   - In the magazine section the pictures should be bigger, the same size as in the editor's pick section.
   - In the newsletter section you display the image when in the original web it does not. Please remove it.
   - You got some extra margin in both sides of the page, please make it smaller.
- Small screens
   - The text in the newsletter section should be aligned centre, you got it on the left some. Please, change it

[OPTIONAL] 
Since this is a project to learn to use bootstrap, maybe you can try to reduce your CSS file a bit and use more bootstrap classes instead.
You can  use the main tag for the dominant content of the body of the document

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes Required :traffic_light: :recycle: 

### Hi @guillainbisimwa & @ositaadimma  :wave: 

This is an awesome job, you're a few clicks to getting it approved :muscle: 
You've carefully made the changes raised by the previous TSE. However, some changes are still required before this project can be approved.

Please make the changes below and make another review request.

- In the `In the magazine` section, the title of the articles are written in Sentence Case in the original page while they're written in Upper Case in your clone page. Kindly fix it.

Your page: 
y


Original Page:
o

- In small screen sizes, the text in the newsletter section is aligned to the left while the original page is aligned to the center. Kindly fix it.

Your page:
y

Original page:
o

* * *
_I'm @taofeekolalere on slack_
**Happy Coding :man_technologist:**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave :wave: Hello team! amazing job! the page is looking great!

Things to notice:
:100: Great looking site! :sunglasses:
:100: Improved readability with 2 space indentation in HTML and CSS
:100: Use of Bootstrap Framework
:100: Nice scrolling effect with sticky property.

The site looks pretty similar to the original, just few fixes related to the previous TSE required changes.

### [Mandatory]

- [ ] **Project not passing linter Checks.** Please submit again until all linter checks are green :heavy_check_mark: 

- [ ] The Newsletter section still doesn't look like the original website
image

- [ ] Subscription and Newsletter have rendering issues
image
image

- [ ] Use ```footer``` semantic HTML tag instead of ```div``` tag for the footer of the page

- [ ] The ```footer``` should be outside the ```main``` section wrapper

## Keep the great work :muscle:, and Happy Coding :smiley: :computer:!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2462 @@
...
+                    class="newsletter-form form-control"
+                    value="Free Sign Up"
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- footer section -->
+        <div class="footer color-white container-fluid">
```
Comment (jubaan) [inline index.html]: - [x] Change the ```div``` opening and closing tag for ```footer```semantic HTML tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2462 @@
...
+            <a
+              class="p-2 border-right-1 no-border-right"
+              rel="noopener"
+              href="#"
+              >Consent preferences</a
+            >
+          </div>
+        </div>
+      </main>
+    </div>
```
Comment (jubaan) [inline index.html]: - [x] Move the footer between the closing ```main``` tag and the closing ```div``` tag

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,
**Your page is impressive. However, afew fixes are needed:**

# General Requirements:

1). **Some topics on the middle navbar, at 992px width, are not displaying properly.**

techdeb_headline.

2). **There is information that is missing at the top leftmost section of your navbar.**
original |  clone
--------- | --------
original  | clone

3). **As requested by the previous TSE, please use a footer tag for your footer section and create it outside the main tag.**

## Goodluck! 😃.

### PS: Incase you want to reach out, please send me a message on slack(@Rose Wanjohi)

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS  💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**